src/server/api/geo/getForestAgreementLayer.ts

```diff
@@ -6,33 +6,54 @@ import { ForestSource } from '@meta/geo'
 import { GeoController } from '@server/controller/geo'
 import Requests from '@server/utils/requests'
 
-export const getForestAgreementLayer = async (req: Request, res: Response) => {
+export const getForestAgreementLayer = async (
+  req: Request<
+    { countryIso: CountryIso },
+    never,
+    never,
+    { gteAgreementLevel: number; gteHansenTreeCoverPerc: number; layer: Array<string>; opacity: number }
```

Comment:
Is is better to defiine layer as `Array<ForestSource` ? Then you dont need to cast it later to `unknown` and `ForestSource`

Reply:
Yes, changed to Array<ForestSource>